addons/core/functions/fnc_loadoutReplaceProcess.sqf

```diff
@@ -0,0 +1,49 @@
+/*
+ 	Name: TFAR_fnc_loadoutReplaceProcess
+
+ 	Author(s):
+		zgmrvn
+
+ 	Description:
+ 		iterates through player's loadouts and replaces actual radios with prototypes
+
+	Parameters:
+		Nothing
+
+ 	Returns:
+		Nothing
+
+ 	Example:
+		[] call TFAR_fnc_loadoutReplaceProcess;
+*/
+
+private _loadouts = profileNamespace getVariable ["bis_fnc_saveinventory_data", []];
+
+// _loadouts is an associative array [loadoutName, loadoutContent, loadoutName, ...], so we have to skip the name in our loop
+for [{private _i = (count _loadouts) - 1}, {_i > 0}, {_i = _i - 2}] do {
+	_inventory = _loadouts select _i;
+
+	// iterate through each container
+	{
+		_content = _x;
+
+		// iterate through each item of the container
+		{
+			_class = ConfigFile >> "CfgWeapons" >> _x;
+
+			// if the item is an actual radio, not a radio prototype nor common item
+			if ((isClass _class) && (isNumber (_class >> "tf_radio"))) then {
+				// find his parent prototype
+				_parent = ([_class, true] call BIS_fnc_returnParents) select 1;
```

Comment:
Only use that to find whats the parent.. as a replacement of calling "BIS_fnc_returnParents select 1" because that will just return the first Parent. And in my planned changes one would have to use the second parent.